Sources/SQLite/Core/Statement.swift

```diff
@@ -208,7 +208,10 @@ public protocol FailableIterator : IteratorProtocol {
 
 extension FailableIterator {
     public func next() -> Element? {
-        return try! failableNext()
+        if let next = try? failableNext() {
```

Comment:
You could simply write `return try? failableNext()` instead of `return try! failableNext()`. There's no need to use `if let` here.

Reply:
No this raises `Value of optional type 'Self.Element??' not unwrapped; did you mean to use 'try!' or chain with '?'?`